simple-user-login-system/run.py

```diff
@@ -0,0 +1,25 @@
+from app import create_app
+from flask import Flask
+import os
+from dotenv import load_dotenv
+load_dotenv()
+
+from app.views.user import user_bp
+
+
+
+
+app = create_app()
+app.register_blueprint(user_bp)
+
+env=os.getenv("ENV")
+print(f'you are working on {env} environment')
+print(app.config['DB'])

```

Comment:
Please remove these print calls

Reply:
hoo! sure, i forget to remove it i was actually using it to check if my application working environment is changing when i switch it in .env. let me remove it.